src/containers/DocumentsList/ChooseDocumentToCreateModal/index.tsx

```diff
@@ -34,6 +34,7 @@ export const ChooseDocumentToCreateModal = (props: Props) => {
                     'subject-type': subjectType ? [subjectType] : [],
                     _sort: 'title',
                     ...(context ? { context } : {}),
+                    status: 'active',
```

Comment:
@alexlipovka context should always be the last to override all other parameters.

Reply:
@ir4y got it!